### ISSUE TITLE
add list-footer to MaterialSelect and MaterialAutoSuggestInput

### ISSUE THREAD

**Issue by jodinathan** [dart-lang/angular_components#186]
Hi,

Could you add a ```<ng-content selector="[list-footer]"></ng-content>``` to the ```<material-list><div *ngFor="..."></div></material-list>``` within ```MaterialSelect``` and ```MaterialAutoSuggestInput```  templates?

Would be like:

```
<material-list>
  <ng-content selector="[list-header]"></ng-content>
  <div *ngFor="..."></div>
  <ng-content selector="[list-footer]"></ng-content>
</material-list>```

this would help so much to paginate stuff.
I am building right now an infinity async list with MaterialAutoSuggestInput and this would make stuff much simpler =]

**Comment by cissyshi** [comment]
MaterialAutoSuggestInput already supports projecting content below the material list through the [footer] selector. We chose to do it this way so when the list is long you can still see the footer while the list scrolls.

**Comment by jodinathan** [comment]
yeah, I know about that footer, however, that does not help infinity scrolling because it is not inside the same scrollable area.
from what I've seen on infinity scroll implementations, they normally add one dull item to the end of the list, or use the last optionitem, and once it is seen (ie. are eye visible to the user), it starts an ajax call to the server, fetching some more items, and appending them to the select's list.
